### ISSUE TITLE
Waiting for grain open blocks DDP methods

### ISSUE THREAD

**Issue by kentonv** [sandstorm-io/sandstorm#1139]
While waiting for a grain to open (while the spinner displays), new DDP method calls and publishes are blocked, meaning if I switch back to the app/grain tabs and click around I get blocked on "loading" screens. We probably just need a `this.unblock()` somewhere.

**Comment by ar-jan** [comment]
Is this issue what leads to e.g. an Etherpad grain without the app's toolbar, if you clicked e.g. the Grains list while it was loading?

**Comment by kentonv** [comment]
@ar-jan I don't think so. I'm not familiar with that issue, and I can't seem to reproduce it. Could you file a new issue and include screenshots?

**Comment by ar-jan** [comment]
Maybe it's browser specific? I've filed #1473.
